apis/projectcontour/v1alpha1/configuration.go

```diff
@@ -0,0 +1,20 @@
+// Copyright Project Contour Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+// ContourConfig represents a configuration of a Contour controller.
+// It contains most of all the options that can be customized, the
+// other remaining options being command line flags.
+type ContourConfig struct {
+}
```

Comment:
I would name this `ContourConfigurationSpec` and move it into the `contourconfig.go` file to keep it all together.

Reply:
👍